### ISSUE TITLE
Multiple-select in grid (using ctrl) doesn't work on Mac

### ISSUE THREAD

**Issue by fabien** [syncfusion/ej2-javascript-ui-controls#37]
I can't seem to get multiple (discontinuous) selection inside the grid to work. This affects probably other list-like controls as well, as control-click interferes with the Mac default contextual menu display.

**Comment by Madhust** [comment]
@fabien  We have analyzed the reported problem. We could see that the ctrl+left click in MAC OS is not handled, which is why multi selection using ctrl+left click is not working. So we suggest you to use the use the “alt+left click” combination to achieve the multi selection in MAC OS, by adding the below codes to your application. Here we have bound the click event and set the value for the private property “enableSelectMultiTouch” based on the “altKey” value from click event handler, please refer the code example below,

```
  <script>
        var data = new ej.data.DataManager(window.employeeData).executeLocal(new ej.data.Query().take(15));
        var grid= new ej.grids.Grid({
            dataSource: data,
            selectionSettings: { type: 'Multiple' },
            ...
        });
        grid.on('click',function (e){
            var grid = document.getElementById('Grid').ej2_instances[0];
            grid.selectionModule.enableSelectMultiTouch = e.altKey;
        },grid);
        grid.appendTo('#Grid');
    </script>
```
We have also attached a sample for your convenience. Please download the sample from the link below,
Sample : http://www.syncfusion.com/downloads/support/directtrac/general/ze/to_customer845327907

Please get back to us if you need further assistance.

**Comment by fabien** [comment]
Thank you, that looks like a solution indeed! Is there a reason why this won't be made the default behavior in the Grid component(s)?

**Comment by Jeevakanth** [comment]
Hi Fabien,

We have checked your query and logged “Multi selection is not working properly in MAC OS” as a bug and it will be included in the upcoming patch release . Until then we appreciate your patience.

You can now track the current status of your request, review the proposed resolution timeline, and contact us for any further inquiries through the below link: 
https://www.syncfusion.com/feedback/6438/multiselection-is-not-handled-in-mac-os

Please get back to us if you need further assistance.

Regards,
Jeevakanth SP.

**Comment by jmfarook** [comment]
Hi All,

This issue ”MultiSelection is not working properly in Mac OS” has been included from our Volume 2, 2019 beta release v17.2.0.28-beta. But currently , We are glad to announce that our Essential Studio 2020 Volume 2 release v18.2.0.44 . So please upgrade to this version to resolve the issue. It is available for download under the following link.

https://www.syncfusion.com/forums/155798/essential-studio-2020-volume-2-release-v18-2-0-44-is-available-for-download


Regards,
J Mohammed Farook

**Comment by ajithr** [comment]
We are closing this issue since there is no recent activity. Feel free to reopen the issue and update your comments for the reinvestigation.
